app/Repositories/Contracts/AbstractRepository.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: quantien
+ * Date: 11/6/18
+ * Time: 4:57 PM
+ */
+
+namespace App\Repositories\Contracts;
+
+interface AbstractRepository
+{
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: PSR2.Files.EndFileNewline.NoneFound</br>**Severity**: error</br>**File**: app/Repositories/Contracts/AbstractRepository.php</br>**Line**: 14</br>Expected 1 newline at end of file; 0 found